lib/targets/cuda/malloc.cpp

```diff
@@ -158,12 +150,12 @@ namespace quda
 
     a.size = size;
 
-#if (CUDA_VERSION > 4000)                                                                                              \
-  && 0 // we need to manually align to page boundaries to allow us to bind a texture to mapped memory
+#if 0
```

Comment:
can we remove this?

Reply:
I left the `if 0` path for future work where we may want to switch to this and remove the alignment.  I think the else path was only there because of texture requirements, and we should be able to remove it.  But this is left for future work and out of scope for this PR.